mipi_detect_node:
  ros__parameters:
  multi_threading: False
  crop_edges: False
  crop_h_ratio: 0.8
  crop_resize: False
  crop_w_ratio: 1
  yolo_detect_config_file: "config.json"
  pushlisher_node_name: "yolo_detections"
  publisher_img_topic_name: "image_raw"

servo_node:
  ros__parameters:
  queue_size: 2
  img_height: 1080
  subscription_topic: "/yolo_detections"

track_deepsort:
  ros__parameters:
  target_class: "person"
  score_threshold: 0.25
  max_age: 30
  n_init: 3
  max_cosine_distance: 0.2
  use_embedder: True
  embedder_half: True
  image_topic: "/image_raw"
  yolo_msg: "yolo_detections"
  debug_log: False